packages/primeng/src/scroller/scroller.ts

```diff
@@ -756,36 +786,36 @@ export class Scroller extends BaseComponent implements OnInit, AfterContentInit,
 
     scrollInView(index: number, to: ScrollerToType, behavior: ScrollBehavior = 'auto') {
```

Comment:
Real type of `index` is `number | [number, number]`, it would be nice to stick to only one type of the union - the tuple, it would cover all cases (all orientations), but would introduce more breaking changes.